File: imagem.bar.francisco.domain/DTO/Account/FluxoDeCaixaDespesa.cs
using imagem.bar.francisco.domain.Interface.Repository;
using System;
using System.Collections.Generic;
using System.Text;

namespace imagem.bar.francisco.domain.DTO.Account
{
    public class FluxoDeCaixaDespesa : AbstractEntity
    {
        public Guid FluxoDeCaixaId { get; set; }
        public Guid DespesaId { get; set; }

        public virtual FluxoDeCaixa FluxoDeCaixa { get; set; }
        public virtual Despesa Despesa { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/account/fluxo-de-caixa-despesa.ts
import { Despesa } from './despesa';
import { AbstractEntity } from "../abstract-entity";
import { FluxoDeCaixa } from './fluxo-de-caixa';

export class FluxoDeCaixaDespesa extends AbstractEntity{
  public  fluxoDeCaixaId: string;
  public  despesaId: string;

  public fluxoDeCaixa: FluxoDeCaixa;
  public despesa: Despesa;
}
